.eslintrc.js

```diff
@@ -3,14 +3,13 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 2022,
   },
   plugins: ['prettier'],
   extends: ['eslint:recommended', 'plugin:n/recommended', 'plugin:prettier/recommended'],
   env: {
     browser: false,
     node: true,
-    es6: true,
```

Comment:
Per the docs, this overrides the parserOptions and sets it back to version 6 instead of the latest: https://eslint.org/docs/latest/use/configure/language-options